packages/wp-story-editor/src/components/documentPane/publish/publish.js

```diff
@@ -414,6 +448,39 @@ function PublishPanel({ nameOverride }) {
             </LabelWrapper>
           </DropdownWrapper>
         </HighlightRow>
+        {improvedAutosaves && revisionCount >= 1 ? (
+          <RevisionsWrapper>
+            <LabelWrapper>
```

Comment:
`LabelWrapper` by default has `height: 40px` so that the publisher logo & poster image setting are more nicely aligned.

We don't need this height though for revisions.

Let's create a new `RevisionsLabelWrapper` or so without that height.

Reply:
Removed the `LabelWrapper` --- not really needed.